### ISSUE TITLE
Screen effects timing off by one frame

### ISSUE THREAD

**Issue by fmatthew5876** [EasyRPG/Player#1546]
Not sure if this really matters, but our timing for tint is slightly off.

Create an event that does this. 

```
Tint screen 5.0s, No Wait
Wait 2.5s
OpenSaveMenu
```

In RPG_RT:
```
0x01 (size: 2, pos: 0xee): tint_finish_red
  150
0x02 (size: 2, pos: 0xf2): tint_finish_green                                         
  150
0x03 (size: 2, pos: 0xf6): tint_finish_blue                                          
  150
0x04 (size: 1, pos: 0xfa): tint_finish_sat
  50
0x0b (size: 8, pos: 0xfd): tint_current_red                                          
125
0x0c (size: 8, pos: 0x107): tint_current_green                                       
125
0x0d (size: 8, pos: 0x111): tint_current_blue                                        
125
0x0e (size: 8, pos: 0x11b): tint_current_sat                                         
75
0x0f (size: 2, pos: 0x125): tint_time_left
  150
```

In Player
```
0x01 (size: 2, pos: 0xee): tint_finish_red
  150
0x02 (size: 2, pos: 0xf2): tint_finish_green
  150
0x03 (size: 2, pos: 0xf6): tint_finish_blue
  150
0x04 (size: 1, pos: 0xfa): tint_finish_sat
  50
0x0b (size: 8, pos: 0xfd): tint_current_red
125.167
0x0c (size: 8, pos: 0x107): tint_current_green
125.167
0x0d (size: 8, pos: 0x111): tint_current_blue
125.167
0x0e (size: 8, pos: 0x11b): tint_current_sat
74.8333
0x0f (size: 2, pos: 0x125): tint_time_left                                           
  149
```

**Comment by fmatthew5876** [comment]
Here is a better test

```
Tint 5.0s, No Wait
OpenSaveMenu
```

RPG_RT
```
Info: Loading RPG_RT database `/home/matt/Documents/RPG Maker 2000/Project2/Save01.lsd'...                                                                                   
0x01 (size: 2, pos: 0xed): tint_finish_red
  150
0x02 (size: 2, pos: 0xf1): tint_finish_green
  150
0x03 (size: 2, pos: 0xf5): tint_finish_blue
  150
0x04 (size: 1, pos: 0xf9): tint_finish_sat
  50
0x0f (size: 2, pos: 0xfc): tint_time_left
  300

```

Player
```
0x01 (size: 2, pos: 0xee): tint_finish_red
  150
0x02 (size: 2, pos: 0xf2): tint_finish_green
  150
0x03 (size: 2, pos: 0xf6): tint_finish_blue                                                                                                                                  
  150                                                                                                                                                                        
0x04 (size: 1, pos: 0xfa): tint_finish_sat
  50
0x0b (size: 8, pos: 0xfd): tint_current_red
100.167
0x0c (size: 8, pos: 0x107): tint_current_green
100.167
0x0d (size: 8, pos: 0x111): tint_current_blue
100.167
0x0e (size: 8, pos: 0x11b): tint_current_sat
99.8333
0x0f (size: 2, pos: 0x125): tint_time_left
  299
```

**Comment by fmatthew5876** [comment]
Bug also exists for flash and shake.

**Comment by Ghabry** [comment]
This is resolved by the "Update X before map" in #1545 , #1548 and #1549
